plugin/src/makeWorklet.ts

```diff
@@ -319,6 +319,7 @@ function makeArrayFromCapturedBindings(
   fun: NodePath<WorkletizableFunction>
 ) {
   const closure = new Map<string, Identifier>();
+  const isLocationSet = new Map<string, boolean>();
```

Comment:
not really happy about this name, first I thought it's a `Set` (maybe it could be a set since we only assign booleans?)

Reply:
I think it was a set originally but then I changed it to the map and didn't update the name. I think the code is simpler with a Map, but if you prefer a Set I'm happy to listen to suggestions.

Reply:
Sounds good